getting-started.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: home
+---
+
+{:.cryostat-heading-1}
+Cryostat 1.0.0
+
+Create and manage JDK Flight Recordings to troubleshoot your containerized Java applications
+
+{:.cryostat-heading-2}
+Now available for Kubernetes 1.19+ and OpenShift 4.6+
+
+## Getting Started
+This section explains some common Cryostat features by example, illustrating
+common actions and workflows of interest and why they are useful.
+
+* auto-gen TOC:
+{:toc}
```

Comment:
I noticed this TOC includes the Getting Started section itself which seems a bit odd. Any ideas? Maybe should be s/##/# above? I think ideally it would have one entry at the moment: Start/Stop a recording

Reply:
The `{:toc}` "directive", or whatever it's called, grabs any Markdown header and puts it into the table. Using `#`, `##`, `###`, etc. actually makes a nested list for the TOC.

However, if I use HTML directly and replace `##` with `<h2>`, then it gives the same rendered appearance but without the header being picked up and included in the `{:toc}`.

Reply:
> The `{:toc}` "directive", or whatever it's called, grabs any Markdown header and puts it into the table. Using `#`, `##`, `###`, etc. actually makes a nested list for the TOC.
> 

Ah figures.

> However, if I use HTML directly and replace `##` with `<h2>`, then it gives the same rendered appearance but without the header being picked up and included in the `{:toc}`.

Nice thanks!